Code/Tools/SceneAPI/SceneUI/SceneWidgets/ManifestWidgetPage.h

```diff
@@ -115,6 +115,7 @@ namespace AZ
                 size_t m_capSize;
                 QString m_helpUrl;
                 QMenu* m_editMenu;
+                bool m_scrollToBottomQueued = false;
```

Comment:
This should be an atomic_bool
```suggestion
                AZStd::atomic_bool m_scrollToBottomQueued = false;
```

Reply:
Happy to change it, but why does it need to be atomic? What other threads are setting this other than the main thread?

Reply:
Well I would believe this was running on a second thread based on the single shot call above,

Reply:
AFAIK, QT timers run on the thread they were queued from ( https://doc.qt.io/qt-6/qtimer.html#details ). I believe this will currently always run on the main thread. I don't think we run Qt event loops on any other threads.

Reply:
Ok, as long as the code is not meant to be run in a multithread context, then it is fine to be a regular bool